app/components/button/component.tsx

```diff
@@ -70,7 +70,7 @@ function buildClassName({
   theme,
 }) {
   return cx({
-    'group relative flex items-center justify-center rounded-4xl focus:outline-none': true,
+    'group relative flex items-center justify-center rounded-4xl transition focus:outline-none': true,
```

Comment:
What if we use `transition-colors` => https://tailwindcss.com/docs/transition-property#class-reference. If we only want to animate colors and backgrounds I think it has better performance. TBH, we won't see any difference, but...